app/src/main/java/com/triviagenai/triviagen/trivia/presentation/triviagame/TriviaGameScreen.kt

```diff
@@ -0,0 +1,66 @@
+package com.triviagenai.triviagen.trivia.presentation.triviagame
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.AbsoluteRoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.dimensionResource
+import com.triviagenai.triviagen.R
+import com.triviagenai.triviagen.core.presentation.TriviaGenScaffold
+import com.triviagenai.triviagen.trivia.data.repository.DummyTriviaRepositoryImpl
+import com.triviagenai.triviagen.trivia.domain.model.TriviaQuestion
+
+@Composable
+fun TriviaGameScreen() {
+    val fakeData =
+        DummyTriviaRepositoryImpl()
+            .fetchTriviaQuestions("what?")
+            .collectAsState(
+                initial = listOf(
+                    TriviaQuestion(
+                        "null",
+                        listOf("1", "2", "3", "4"),
+                        2
+                    )
+                )
+            ).value
+
+    TriviaGenScaffold {
+        Column(
+            modifier = Modifier
+                .fillMaxSize(),
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Text(
+                text = fakeData[0].question,
+                modifier = Modifier
+                    .padding(dimensionResource(id = R.dimen.padding_medium))
+            )
+
+            fakeData[0].options.forEachIndexed { index, trivia ->
+                Button(
+                    onClick = {
+
+                    },
+                    shape = AbsoluteRoundedCornerShape(dimensionResource(id = R.dimen.rounded_corner)),
+                    modifier = Modifier
+                        .padding(dimensionResource(id = R.dimen.padding_small))
+                        .width(dimensionResource(id = R.dimen.element_xlarge))
+                ) {
+                    Text(
```

Comment:
Is the Option text vertically centered?

Reply:
It's complicated. The small letters like ,,a" or ,,n" are in the middle, so from top edge of the letter is the same distance to the button border as from the bottom of letter. But in bigger letters like ,,b" or ,,V" it is different. They are higher, so if they want to be in the middle, as it is in the lower letters case, they would be a little below lower letters. And in my opinion it is some kind of viusal effect, that bigger letters generete and that is why it seems to be uncentered.

Green lines are the same height.
![obraz](https://github.com/sidcgithub/ai-trivia-app-android/assets/91783342/00cb7f2f-f5b6-4212-8dc1-6bb6cfe06ec2)

Reply:
Hey makes sense.